tool/site/server/src/routes/auth/login.ts

```diff
@@ -0,0 +1,43 @@
+import Hapi from "@hapi/hapi"
+import createOAuthClient from "../../oauthUtils"
+import {setSessionValue} from "../../services/session"
+import {createOAuthState} from "../helpers"
+
+interface LoginInfo {
+  accessToken: string
+  authLevel: "user" | "system"
+  authMethod: string
+}
+
+export default {
+  method: "POST",
+  path: "/login",
+  options: {
+    auth: false
+  },
+  handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
+
+    const loginInfo = request.payload as LoginInfo
+
+    setSessionValue(`auth_level`, loginInfo.authLevel, request)
+    setSessionValue(`auth_method`, loginInfo.authMethod, request)
+
+    if (process.env.ENVIRONMENT?.endsWith("sandbox")) {
+      // Local
+      return h.response({redirectUri: "/callback"}).code(200)
+    }
+
+    if (loginInfo.authLevel === "system") {
+      // todo (unattended auth)
```

Comment:
unattended is handled in a different route?

Reply:
currently, will be merging into the same route, as they both "log you in"